cypress/integration/lock.spec.js

```diff
@@ -14,7 +14,7 @@ context("lock screen", () => {
     cy.pick("unlock-button").should("exist");
     cy.pick("passphrase-input").type("wrong-pw");
     cy.pick("unlock-button").click();
-    cy.contains(/Could not unlock the session: Passphrase incorrect/).should(
+    cy.contains(/Could not unlock the session: incorrect password/).should(
```

Comment:
```suggestion
    cy.contains(/Could not unlock the session: incorrect passphrase/).should(
```

Reply:
Already fixed in https://github.com/radicle-dev/radicle-upstream/commit/751ddd213d26ef187ab6d84ea8b83ef22562df21